apps/meteor/app/api/server/v1/moderation.ts

```diff
@@ -113,6 +161,61 @@ API.v1.addRoute(
 	},
 );
 
+// api endpoint to get user reports of a userid
+
+API.v1.addRoute(
+	'moderation.user.reportsByUserId',
+	{
+		authRequired: true,
+		validateParams: isGetUserReportsParams,
+		permissionsRequired: ['view-moderation-console'],
+	},
+	{
+		async get() {
+			const { userId, selector = '' } = this.queryParams;
+			const { sort } = await this.parseJsonQuery();
+			const { count = 50, offset = 0 } = await getPaginationItems(this.queryParams);
+
+			const user = await Users.findOneById<IUser>(userId, {
+				projection: {
+					_id: 1,
+					username: 1,
+					name: 1,
+					avatarETag: 1,
+					active: 1,
+					roles: 1,
+					emails: 1,
+					createdAt: 1,
+				},
```

Comment:
what defined these fields need to be exposed? also, we have view-other-user-info, IMO we should respect that permission here too.

Reply:
Thank you for the comment, Deb!

> what defined these fields need to be exposed? 

Nothing precisely; it is just the info we must show on the Context Bar.

> we have view-other-user-info, IMO we should respect that permission here too.

Sure, we could do that, but should we? Because viewing the Moderation Console already requires permission, if an admin doesn't want the reported user details to be viewed by specific role users, they won't give permission to view the Moderation Console. 
Additionally, I think the details on MC, apart from the User info, are already quite sensitive. Thoughts?